tools/rts_verifier/basic_types.py

```diff
@@ -0,0 +1,58 @@
+from enum import Enum
+
+class PrepareState(Enum):
```

Comment:
Can you auto-pull these from somewhere? Or add a script to the `dist` directory and `s_all` that checks to ensure they remain consistent with the ones in WiredTiger?

Reply:
That's a fair point - it wasn't as difficult as I thought, done in 7c2702a.